React/Modules/RCTStatusBarManager.m

```diff
@@ -103,6 +103,26 @@ - (void)applicationWillChangeStatusBarFrame:(NSNotification *)notification
   }]);
 }
 
+RCT_EXPORT_METHOD(getStatusBarOrientation:(RCTPromiseResolveBlock)resolve
+                                   reject:(RCTPromiseRejectBlock)reject)
+{
+  NSDictionary *orientations = @{
```

Comment:
Thanks. I think RCTConvert is only helpful for setter methods. I could't use it to _reverse_ convert without re-declaring a dictionary.